src/barril/units/_array.py

```diff
@@ -246,6 +246,8 @@ def __iter__(self):
     def __truediv__(self, other):
         return self._DoOperation(self, other, "Divide")
 
+    __floordiv__ = __truediv__
```

Comment:
This doesn't seem right, why don't we support `floordiv`?

Reply:
Hm, not sure as well. Now that you've asked, I'll add the support.

Reply:
BTW I'm adding the support for `Scalar` as well (they do not have it)